distributions/LibreELEC/options

```diff
@@ -238,3 +238,8 @@
 # Space separated list is supported,
 # e.g. ADDITIONAL_PACKAGES="PACKAGE1 PACKAGE2"
   ADDITIONAL_PACKAGES=""
+
+# u-boot device-tree overlays to enable:
+# Whitespace separated list without path and extension,
+# e.g. ENABLED_DTOVERLAYS="sun8i-h2-plus-bpi-m2-zero-ethernet"
+  ENABLED_DTOVERLAYS=""
```

Comment:
This section needs to be moved to project or device level options as we will not have any universal overlays (for technical reasons). I suspect device/options is most appropriate.